src/components/ClipboardAwarePasteButton.tsx

```diff
@@ -30,14 +39,54 @@ export default function ClipboardAwarePasteButton({
     return null
   }
 
+  // Debounce onPress event so that it is called once on trigger and
+  // consecutive calls in given period are ignored.
+  const debouncedOnPress = debounce(
+    async () => {
+      await onPressInternal()
+    },
+    BUTTON_TAP_DEBOUNCE_TIME,
+    DEBOUNCE_OPTIONS
+  )
+
   return (
-    <Button
-      text={t('paste')}
-      type={BtnTypes.ONBOARDING_SECONDARY}
-      rounded={false}
-      size={BtnSizes.FULL}
-      onPress={onPressInternal}
-      testID={'PasteButton'}
-    />
+    <View style={styles.container}>
+      {/* these Views cannot be combined as it will cause ripple to not respect the border radius */}
+      <View style={styles.containRipple}>
```

Comment:
Is this required? I am pretty sure we have rounded touchables working with a single View in other places. cc @MuckT tagging you since you've done a lot of work related to ripples on android

Reply:
The touchable shouldn't be rounded, removed it in [0ec330a](https://github.com/valora-inc/wallet/pull/5628/commits/0ec330a29c9ba037741e99d50e72a18f0c27f81a) but can add it back if it will cause android issues.